package.json

```diff
@@ -235,12 +235,12 @@
     "test": "yarn test:types && yarn test:server && yarn test:app",
     "test:app": "jest --forceExit",
     "test:server": "cd server && yarn test",
-    "test:types": "tsc --noEmit && cd server && tsc --noEmit",
+    "test:types": "tsc --project tsconfig.build.json --noEmit && cd server && tsc --noEmit",
     "translate:extract": "i18next-scanner 'src/{page,script}/**/*.{js,html,htm}'",
     "translate:merge": "formatjs extract --format './bin/translations_extract_formatter.js' --out-file './src/i18n/en-US.json' './src/script/strings.ts'",
     "translate:upload": "yarn translate:merge && ts-node ./bin/translations_upload.ts",
     "translate:download": "ts-node ./bin/translations_download.ts",
-    "betterer": "betterer",
+    "betterer": "betterer precommit",
```

Comment:
is there a use case for just betterer? i imagine for testing you don't want to run the precommit version

Reply:
i don't see one. Betterer without writing the file is kind of redundant. I would rather run yarn tsc --noEmit in that case